### PR TITLE
ENH: Display markups over model for draw tube and surface cut

### PR DESCRIPTION
**Pull request by adamaji**
This exposes a function that toggles a setting which controls the order the markup and DrawTube model nodes are added to the scene. (https://discourse.slicer.org/t/how-to-change-display-layer-of-markups/8960) 

@jamesobutler can you take a look as well?

### CODE REVIEW

**Comment by lassoan** [comment]
Do you need the toggle feature or you always want to have a specific order? If you want to have a specific order then don't make it dynamic just change the code to always display items in that preferred order.

**Comment by adamaji** [comment]
I think the toggle feature is nice to have as a user preference -- control points over the tube make the editability more clear but obscures some parts of the tube, control points underneath the tube is perhaps less clear but allows full view of the tube. I wasn't sure if the initial ordering was done for a reason.

File: SegmentEditorDrawTube/SegmentEditorDrawTubeLib/SegmentEditorEffect.py [hunk 1]
```diff
@@ -23,6 +23,8 @@ def __init__(self, scriptedEffect):
     self.segmentObserver = None
     self.buttonToInterpolationTypeMap = {}
 
+    self.displayMarkupOverModel = True
```
Comment (adamaji) [inline SegmentEditorDrawTube/SegmentEditorDrawTubeLib/SegmentEditorEffect.py]: this needs to be changed to `displayMarkupUnderModel`

**Comment by lassoan** [comment]
Ok, since dynamic changing of the order is non-essential, please change the implementation to hardcode the order to always show markups on top. Add a comment to the critical part to ensure that developers who change the code in the future not accidently break this mechanism. Thank you!

**Review by jamesobutler — APPROVED** [review]
Thank you for doing this for SurfaceCut as well since it is extremely similar to DrawTube. @lassoan I've tested and this change looks good.

**Comment by lassoan** [comment]
Thank you guys! Merged.

**Comment by adamaji** [comment]
Thanks!

**Pull request merged**